File: Examples/VisualBasic/Data/Handling/AccessingCells/RowAndColumnIndex.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells
Imports System

Namespace Data.Handling.AccessingCells
    Public Class RowAndColumnIndex
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Instantiating a Workbook object
            Dim workbook As New Workbook(dataDir & "book1.xls")

            ' Using the Sheet 1 in Workbook
            Dim worksheet As Worksheet = workbook.Worksheets(0)

            ' Accessing a cell using its name
            Dim cell As Cell = worksheet.Cells(0, 0)

            Dim value As String = cell.Value.ToString()

            Console.WriteLine(value)
            ' ExEnd:1

        End Sub
    End Class
End Namespace

File: Examples/CSharp/Data/Handling/AccessingCells/RowAndColumnIndex.cs
using System.IO;

using Aspose.Cells;
using System;

namespace Aspose.Cells.Examples.Data.Data.Handling.AccessingCells
{
    public class RowAndColumnIndex
    {
        public static void Main(string[] args)
        {
            //ExStart:1
            // The path to the documents directory.
            string dataDir = Aspose.Cells.Examples.Utils.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            //Instantiating a Workbook object
            Workbook workbook = new Workbook(dataDir + "book1.xls");

            //Using the Sheet 1 in Workbook
            Worksheet worksheet = workbook.Worksheets[0];

            //Accessing a cell using its name
            Cell cell = worksheet.Cells[0,0];

            string value = cell.Value.ToString();

            Console.WriteLine(value);
            //ExEnd:1
            
        }
    }
}
